TestVectorsAwsCryptographicMaterialProviders/dafny/KeyVectors/src/KeyDescription.dfy

```diff
@@ -218,8 +218,8 @@ module {:options "-functionSyntax:4"} KeyDescription {
          // SYMMETRIC_DEFAULT is not supported because RSA is asymmetric only
        ]
 
-  type KeyDescriptionVersion = v: nat | KeyDescriptionVersion?(v)  witness 1
-  predicate KeyDescriptionVersion?(v: nat)
+  type KeyDescriptionVersion = v: int | KeyDescriptionVersion?(v)  witness 1
+  predicate KeyDescriptionVersion?(v: int)
```

Comment:
Also meant to mention this in the PR: Dafny master has a change to support `is` runtime tests for more types. But because of that, the `nat` type in the runtimes has additional new methods that generated code depends on. That means with this code, it's impossible to build on Dafny nightlies because there's no corresponding Dafny Java runtime nightly prerelease with those new methods.

The safe workaround is to use `int` as the basetype instead, which is equivalent since the `KeyDescriptionVersion?` predicate already constrains it to positive values anyway.